File: C/if else/endwith5.c
/*Check if a number ends with digit 5*/

#include <stdio.h>

int main() {
    int num;
    
    printf("Enter number:");
    scanf("%d",&num);
    
    if(num%10==5){
        printf("number ends with 5");
    }
    else{
        printf("number does not ends with 5");
    }

}

File: java/if else/endwith5.java
/*#Check if a number ends with digit 5*/

import java.util.Scanner;

class endwith5{
    public static void main(String args[]){
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter a number:");
        int num=sc.nextInt();

        if(num%10==5){
            System.out.println(num+" ends with digit 5");
        }
        else{
            System.out.println(num+" does not ends with digit 5");
        }
    }
}

